content/courses/onchain-development/anchor-pdas.md

```diff
@@ -24,16 +24,14 @@ In this lesson you'll learn how to work with PDAs, reallocate accounts, and
 close accounts in Anchor.
 
 Recall that Anchor programs separate instruction logic from account validation.
-Account validation primarily happens within structs that represent the list of
-accounts needed for a given instruction. Each field of the struct represents a
-different account, and you can customize the validation performed on the account
-using the `#[account(...)]` attribute macro.
+Account validation happens in structs that list the accounts needed for an
+instruction. Each field in the struct represents an account, and you can
+customize the validation using the `#[account(...)]` attribute macro.
 
-In addition to using constraints for account validation, some constraints can
-handle repeatable tasks that would otherwise require a lot of boilerplate inside
-our instruction logic. This lesson will introduce the `seeds`, `bump`,
-`realloc`, and `close` constraints to help you initialize and validate PDAs,
-reallocate accounts, and close accounts.
+In addition to validating accounts, some constraints can automate tasks that
+would otherwise require repetitive code in our instructions. This lesson will
+cover the `seeds`, `bump`, `realloc`, and `close` constraints to help you easily
+handle PDAs, reallocate space, and close accounts.
```

Comment:
I can already tell here this PR is going to be a massive improvement. The writing style is much more specific and to-the-point.👍

Reply:
thanks